bindata/assets/alerts/cpu-utilization.yaml

```diff
@@ -12,6 +12,7 @@ spec:
             summary: >-
               CPU utilization across all three control plane nodes is higher than two control plane nodes can sustain; a single control plane node outage may
               cause a cascading failure; increase available CPU.
+            runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/cluster-kube-apiserver-operator/ExtremelyHighIndividualControlPlaneCPU.md
```

Comment:
I think it is fine to share the runbook between `ExtremelyHighIndividualControlPlaneCPU` and `HighOverallControlPlaneCPU`, but we should mention `HighOverallControlPlaneCPU` in the runbook which is not the case today.

Reply:
okay, i can add a few lines in the runbook